### PR TITLE
Updating with the mainline branch

### PR DESCRIPTION
**Pull request by jonathan**
risk: low
Currently hitting a problem in rubocop-git related to this: https://github.com/m4i/rubocop-git/issues/13

### CODE REVIEW

**Comment by adamjgray** [comment]
:+1:

File: test/rubocop/git/runner_test.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require_relative '../../test_helper'
+require 'rubocop/git/runner'
+
+describe RuboCop::Git::Runner do
+  it 'exit with violations' do
+    options = RuboCop::Git::Options.new
+    # lib/rubocop/git/runner.rb:14:1: C: Trailing whitespace detected.
```
Comment (jkieffaber) [inline test/rubocop/git/runner_test.rb]: What's up with this comment?  Can we remove it?

**Comment by jkieffaber** [comment]
Looks good to me beyond the random comment.

**Comment by jonathan** [comment]
I'd merge but I don't have write access to the repo.

**Pull request merged**